developer/src/kmc-kmn/src/kmw-compiler/compiler-globals.ts

```diff
@@ -4,7 +4,7 @@ import { KmnCompilerResult } from "../compiler/compiler.js";
 
 export let FTabStop: string;
 export let nl: string;
-export let FCompilerWarningsAsErrors = false;
+export const FCompilerWarningsAsErrors = false;
```

Comment:
hmm, I think this is actually unused ...

Reply:
My project-wide search suggests that it's not referenced by any TS modules.